CMakeLists.txt

```
@@ -37,6 +37,10 @@ mark_as_advanced(HIGHFIVE_HAS_FRIEND_DECLARATIONS)
 
 option(HIGHFIVE_FIND_HDF5 "Find and link with HDF5." On)
 
+set(HIGHFIVE_CMAKE_INSTALL_DIR "share/HighFive/cmake" CACHE STRING
```

Small nitpick: we originally had `lib/cmake/HighFive...`

I personally feel that swapping the order will make the `share` directory a bit more manageable.
```suggestion
set(HIGHFIVE_CMAKE_INSTALL_DIR "share/cmake/HighFive" CACHE STRING
```

---

Come to think of it, `lib` seems more appropriate - that is also where `pkgconfig` canonically seems to expect its library configuration files. The `share/cmake` would also host the CMake internals (at least on my system)

---

Thanks for the comment. I just checked again where cmake looks for config files in calls to `find_package` in config mode, and it seems that it looks in a relatively large number of places: https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure

So I guess `lib/cmake/<name>`  should be fine as well as all the other options that came up as far as I can see. Sorry for making this change too hasty, didn't seem necessary.